Base/home/anon/.config/SystemServer.ini

```diff
@@ -126,3 +131,6 @@ KeepAlive=true
 
 [Terminal]
 WorkingDirectory=/home/anon
```

Comment:
Do we need change `/home/anon` to `/home/%sid`?

Reply:
Nope, that doesn't make sense.
`%sid` is replaced by the session leader id, see the POSIX documentation for what it means.

Though, if you want to implement something like `%user`, feel free to open a PR.

I think that this kind of question is more appropriate on the discord server than on a comment here.